arm-resources/subscriptions/2016-06-01/swagger/subscriptions.json

```diff
@@ -0,0 +1,312 @@
+        "description": "Gets a list of the tenantIds.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/TenantListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Location": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The ID of the resource (/subscriptions/SubscriptionId)."
+        },
+        "subscriptionId": {
+          "type": "string",
+          "description": "The subscription Id."
+        },
+        "name": {
+          "type": "string",
+          "description": "The location name."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The display name of the location."
+        },
+        "latitude": {
+          "type": "string",
+          "description": "The latitude of the location."
+        },
+        "longitude": {
+          "type": "string",
+          "description": "The longitude of the location."
+        }
+      },
+      "description": "Location information."
+    },
+    "LocationListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Location"
+          },
+          "description": "The locations."
+        }
+      },
+      "description": "Location list operation response."
+    },
+    "Subscription": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The ID of the resource (/subscriptions/SubscriptionId)."
+        },
+        "subscriptionId": {
+          "type": "string",
+          "description": "The subscription Id."
+        },
+        "tenantId": {
+          "type": "string",
+          "description": "The tenant Id."
+        },
+        "displayName": {
+          "type": "string",
+          "description": "The subscription display name."
+        },
+        "state": {
+          "type": "string",
+          "description": "The subscription state."
+        },
+        "subscriptionPolicies": {
+          "$ref": "#/definitions/SubscriptionPolicies",
+          "description": "The subscription policies."
+        },
+        "authorizationSource": {
+          "type": "string",
+          "description": "The authorization source of the request. Valid values are one or more combinations of Legacy, RoleBased, Bypassed, Direct and Management. For example, 'Legacy, RoleBased'."
+        }
+      },
+      "description": "Subscription information."
+    },
+    "SubscriptionPolicies": {
+      "properties": {
+        "locationPlacementId": {
+          "type": "string",
+          "description": "The subscription location placement Id."
```

Comment:
what does this mean? The description does not provide any meaningful information. Can you provide a link to the MSDN doc that provides more info?